pkg/virt-controller/watch/migration/migration.go

```diff
@@ -1509,10 +1512,16 @@ func (c *Controller) sync(key string, migration *virtv1.VirtualMachineInstanceMi
 				}
 			} else {
 				log.Log.Object(vmi).V(5).Info("decentralized migration creating target pod in vmi namespace, source pod based on target VMI")
-				// This is a decentralized target, generate the source pod template
-				sourcePod, err = c.templateService.RenderLaunchManifest(vmi)
+				vmiCopy := vmi.DeepCopy()
+				if vmiCopy.Spec.Domain.Devices.TPM != nil {
```

Comment:
Maybe offtopic: What about persistent EFI?

Also, this should only apply if TPM/EFI are persistent, no?

Reply:
Can persistent EFI run into the same issue? Do you need to skip its PVC too?

Reply:
Anything that writes stuff to the backend storage will have this issue. I didn't know it was possible to have persistent EFI. So if that uses the backend storage then yes.

Reply:
I modified the code to also check persistent EFI, looking at the function that check, it checks both TPM and EFI. I also changed it so it uses the same functions to check if TPM/EFI is in use.

Reply:
Yeah, EFI also uses backend storage, and potentially more things in the future... Until now there was only one function that needed to know the list: `IsBackendStorageNeededForVMI()`. If/when we add more uses for backend-storage, it will be really easy to miss this location... Maybe consider adding a `DisableAllBackendStorageForVMI` function in the backendstorage package, next to that other function? At least that way it would still be centralized...